CMR/python/README.md

```diff
@@ -46,6 +68,26 @@ For testing and general exploration it is advised that the User Acceptance Testi
 ### Testing
 Use either `runme.sh -t` or `python3 -m unittest discover`
 
+## Using the runme.sh script
+The runme.sh script documents how to run all the software life cycles for the project like testing, building, and installing. The script takes many flags, the lastest can be viewed by calling `./runme.sh -h`. Below is also a table of these flags. All flags can be called more then once except -h which ends the script.
```

Comment:
Wording, maybe not lifecycle, but stages. Can probably combine and simplify the second and third sentences. "The script may be invoked with the following flags"